static/js/components/dashboard/CourseAction.js

```diff
@@ -112,12 +113,27 @@ export default class CourseAction extends React.Component {
     }
   );
 
+  renderDescriptionForError = R.curry(
```

Comment:
Since we're only using this in one place it seems like we should just move this code below to `the `STATUS_PAID_BUT_NOT_ENROLLED` case. Then we can remove the checks for `contact` and `text`